### PR TITLE
Docker 1.7 /tags/<tag> endpoint

### PR DESCRIPTION
**Pull request by aprajshekhar**
Functionalities implemented:
1. Returns json string containing an object having image id associated with tag name passed as path param.
2. Return 404 if latest tag is not found.

### CODE REVIEW

File: crane/views/v1.py [hunk 1]
```diff
@@ -110,6 +112,30 @@ def repo_tags(repo_id):
     return repository.get_tags_for_repo(repo_id)
 
 
+@section.route('/repositories/<path:repo_id>/tags/latest')
```
Comment (ncdc) [inline crane/views/v1.py]: Is there a reason why the tag is hardcoded to `latest` instead of being a path param?

cc @mhrivnak

**Comment by aprajshekhar** [comment]
Updated as per review comment. As initial discussions did not clarify requirement of /tag/<tag_name> instead of latest for tag, path param was not used. The latest commit uses path param.

File: crane/views/v1.py [hunk 2]
```diff
@@ -110,6 +112,33 @@ def repo_tags(repo_id):
...
+    Returns a json containing an object that has image id associated with the tag
+    latest.
+
+    :param repo_id: unique ID for the repository. May contain 0 or 1 of the "/"
+                    character. For repo IDs that do not contain a slash, the
+                    docker client currently prepends "library/" when making
+                    this call. This function strips that off.
+    :type  repo_id: basestring
+
+    :param tag_name: name of the tag whose associated image ide has to be returned
```
Comment (ncdc) [inline crane/views/v1.py]: typo: ide -> id

**Comment by ncdc** [comment]
@aprajshekhar would you mind updating the PR title, description, and commit message to make it clear this isn't just for `latest`?

File: crane/views/v1.py [hunk 3]
```diff
@@ -110,6 +112,33 @@ def repo_tags(repo_id):
     return repository.get_tags_for_repo(repo_id)
 
 
+@section.route('/repositories/<path:repo_id>/tags/<tag_name>')
+def repo_tags_get_tag(repo_id, tag_name):
+    """
+    Returns a json containing an object that has image id associated with the tag
+    latest.
```
Comment (ncdc) [inline crane/views/v1.py]: s/latest/`<tag_name>`/ ?

**Comment by tombutt** [comment]
are there outstanding issues with this PR that we need to resolve?

**Pull request merged**